README.md

```diff
@@ -141,6 +140,25 @@ code examples in our YARD documentation. To run `doctest` locally:
 bundle exec yard doctest
 ```
 
+### A Note on Testing
```

Comment:
I couldn't figure out a good place in YARD to put this but let me know if there is!